### ISSUE TITLE
aarch64 macOS runner support

### ISSUE THREAD

**Issue by zugazagoitia** [Hakky54/sslcontext-kickstart#461]
[They're now available](https://github.blog/changelog/2024-01-30-github-actions-introducing-the-new-m1-macos-runner-available-to-open-source/) by using the [`macos-14`](https://github.com/actions/runner-images/blob/main/images/macos/macos-14-arm64-Readme.md) OS in GH Actions.

**Comment by Hakky54** [comment]
Amazing news, thank you for sharing this! I am using `macos-latest` option so I expect the macos-14 will be automatically used. However I just tried it out forcing a specific version and the performance looks promising.

By the way, congratulations with merging javalin-ssl with the main repository of javalin. It seems like it got well adopted and accepted!

**Comment by zugazagoitia** [comment]
> Amazing news, thank you for sharing this! I am using `macos-latest` option so I expect the macos-14 will be automatically used. 

Well, actually.... No. The macos-latest is macos-12: https://github.com/actions/runner-images

> By the way, congratulations with merging javalin-ssl with the main repository of javalin. It seems like it got well adopted and accepted!

Thank you! We just moved it so I didn't have to deal with releases on a separate repo, easier to release it as a module.

**Comment by zugazagoitia** [comment]
I see you closed it by merging, but I would recommend having **both** macos-13 and 14, to test against x64 and aarch64 Mac targets.

**Comment by Hakky54** [comment]
Sure, I can adjust that, however what would be the benefit of testing it on two different architectures? Did you faced any issues in the past with this library and javalin or is it just to have a better os/architecture coverage?

**Comment by zugazagoitia** [comment]
yes. I did. Conscrypt/bouncycastle do not support aarch64 for example.... I only find out by some issue.
When dealing with crypto stuff, very dependent on the JDK/native implementation, I've found it a good idea to cover arch/os separately.

**Comment by Hakky54** [comment]
Sounds a good reason, I have added it now
